xtable-core/src/test/java/org/apache/xtable/ITConversionController.java

```diff
@@ -261,6 +281,94 @@ public void testVariousOperations(
     }
   }
 
+  // The test content is the same as testVariousOperations
```

Comment:
Since this is only testing end to end UUID support, we can limit the amount of commits and keep this test smaller

Reply:
> Since this is only testing end to end UUID support, we can limit the amount of commits and keep this test smaller

Thanks for the suggestion, have simplified it in latest commit :)